### PR TITLE
Feature/m0078 ddg 442 add development tools dependency discoverer

### PR DESCRIPTION
**Pull request by tboue**

### CODE REVIEW

File: ApplicationDeveloperGuide/dependencyDiscoverer.rst [hunk 1]
```diff
@@ -0,0 +1,35 @@
+.. _dependencyDiscoverer:
```
Comment (frivieremicroej) [inline ApplicationDeveloperGuide/dependencyDiscoverer.rst:1]: The build has failed https://readthedocs.com/projects/microej-docs/builds/957472/
Please check links or potential warnings

File: ApplicationDeveloperGuide/dependencyDiscoverer.rst [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+	It is a versatile tool and can be used in other contexts, for instance, to list every dependency of a JAR file.
+	
+	It can be used through a command-line interface, with the possibility to output the result in JSON or XML format, allowing an easy scripting process.
+	
+
+Installation
+============
+
+	This tool is available at https://github.com/MicroEJ/Tool-ApiDependencyDiscoverer. A JAR and Windows executable version can be downloaded from the release page.
+	It is also possible to clone the project, open it into MicroEJ's SDK and use it from sources.
```
Comment (frivieremicroej) [inline ApplicationDeveloperGuide/dependencyDiscoverer.rst:21]:
```suggestion
	It is also possible to clone and import the project in the SDK and use it from sources.
```

File: ApplicationDeveloperGuide/dependencyDiscoverer.rst [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+=====================
+Dependency Discoverer
+=====================
+
+
+Introduction
+============
+
+	The Dependency Discoverer is a tool designed to list missing dependencies of a program in the MicroEJ's central repository.
```
Comment (frivieremicroej) [inline ApplicationDeveloperGuide/dependencyDiscoverer.rst:11]: Please reuse the github readme introduction:

Dependency Discoverer is a tool that lists unresolved dependencies (types, methods and fields) of a set of Java ARchive (JAR) files and .class files.

**Review by frivieremicroej — CHANGES_REQUESTED** [review]

File: ApplicationDeveloperGuide/dependencyDiscoverer.rst (hunk 2 above)
Comment (tboue) [reply ApplicationDeveloperGuide/dependencyDiscoverer.rst]: done in e3d0ebe0e67c3bc99a505d4c8883b1ada13edffb

File: ApplicationDeveloperGuide/dependencyDiscoverer.rst (hunk 3 above)
Comment (tboue) [reply ApplicationDeveloperGuide/dependencyDiscoverer.rst]: done in e3d0ebe0e67c3bc99a505d4c8883b1ada13edffb

File: ApplicationDeveloperGuide/dependencyDiscoverer.rst (hunk 1 above)
Comment (tboue) [reply ApplicationDeveloperGuide/dependencyDiscoverer.rst:1]: passed https://readthedocs.com/projects/microej-docs/builds/958390/

File: ApplicationDeveloperGuide/dependencyDiscoverer.rst [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+=====================
+Dependency Discoverer
+=====================
+
+
+Introduction
+============
+
+	MicroEJ Dependency Discoverer is a tool that lists unresolved dependencies (types, methods and fields) of a set of Java ARchive (JAR) files and .class files.
```
Comment (frivieremicroej) [inline ApplicationDeveloperGuide/dependencyDiscoverer.rst:11]:
```suggestion
	Dependency Discoverer is a tool that lists unresolved dependencies (types, methods and fields) of a set of Java ARchive (JAR) files and .class files.
```

**Review by frivieremicroej — APPROVED** [review]

**Pull request merged**